samples/openapi3/client/features/one-of/python-experimental/.travis.yml

```diff
@@ -0,0 +1,17 @@
+# ref: https://docs.travis-ci.com/user/languages/python
+language: python
+python:
+  - "2.7"
+  - "3.2"
+  - "3.3"
+  - "3.4"
+  - "3.5"
```

Comment:
🙏 
```suggestion
```

Reply:
@spacether do we need all these pythons?

🐍

Reply:
Can we keep 3.5?

Reply:
Hm. I do see:

```
$ cat samples/client/petstore/python-experimental/.travis.yml
# ref: https://docs.travis-ci.com/user/languages/python
language: python
python:
  - "3.5"
  - "3.6"
  - "3.7"
  - "3.8"
# command to install dependencies
install:
  - "pip install -r requirements.txt"
  - "pip install -r test-requirements.txt"
# command to run tests
script: pytest --cov=petstore_api
```

Maybe this will be resolved if master gets integrated up into this branch.